node/src/actors/chain_manager/actor.rs

```diff
@@ -252,6 +253,12 @@ impl ChainManager {
                     }
                 }
 
+                // TODO: use conditional hard fork
+                let rescue_committee = RESCUE_COMMITTEE
```

Comment:
This should probably be

```suggestion
                let rescue_committee = get_rescue_committee(get_environment());
```

Otherwise it's not possible to start the node in testnet mode because of "Malformed signing committee"